src/components/screens/admin-vacation-management/EditVacationDialog.tsx

```diff
@@ -67,42 +85,84 @@ const EditVacationDialog = ({
   onSave,
   disableSave
 }: EditVacationDialogProps) => {
+  const currentYear = new Date().getFullYear().toString();
+  const[selectedYear,setSelectedYear] = useState<string>(currentYear);
```

Comment:
Check your biome (linter) config and ensure you have it set to format on save, it looks like it's not formatting at the moment.